tests/components/broadlink/test_init.py

```diff
@@ -85,18 +89,19 @@ def dummy_broadlink():
     now = utcnow()
 
     with patch.object(
-        hass.components.persistent_notification, "async_create"
-    ) as mock_create, patch("homeassistant.components.broadlink.utcnow") as mock_utcnow:
+            hass.components.persistent_notification,
+            "async_create") as mock_create, patch(
+                "homeassistant.components.broadlink.utcnow") as mock_utcnow:
 
         mock_utcnow.side_effect = [now, now + timedelta(20)]
 
-        await hass.services.async_call(DOMAIN, SERVICE_LEARN, {"host": DUMMY_HOST})
+        await hass.services.async_call(DOMAIN, SERVICE_LEARN,
+                                       {"host": DUMMY_HOST})
         await hass.async_block_till_done()
 
         assert mock_device.enter_learning.call_count == 1
         assert mock_device.enter_learning.call_args == call()
 
         assert mock_create.call_count == 1
-        assert mock_create.call_args == call(
-            "No signal was received", title="Broadlink switch"
-        )
+        assert mock_create.call_args == call("No signal was received",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)